pkg/credentialstores/secret_test.go

```diff
@@ -0,0 +1,28 @@
+package credentialstores
+
+import (
+	"io/ioutil"
+	"net/url"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+func TestCredentialsForSecrets(t *testing.T) {
+	data, err := ioutil.ReadFile("test/image-secrets.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var secrets corev1.SecretList
+	if err := json.Unmarshal(data, &secrets); err != nil {
```

Comment:
https://github.com/openshift/library-go/blob/6d930b1c27cce8ffa64ad7982b8bc5314ee569b1/pkg/operator/resource/resourceread/core.go#L28

Reply:
I could not find a function to read a SecretList. Other than that, this code has been replaced by yaml decoding.